python/ray/serve/batching.py

```diff
@@ -36,6 +37,11 @@ class _GeneratorResult:
     next_future: asyncio.Future
 
 
+# Singleton enum used as token to indicate that generator in batch is finished
+class SENTINEL(Enum):
```

Comment:
The user currently has to write `from ray.serve.batching import SENTINEL` to access this object. Should we offer a smoother import path?

Reply:
Resolved with @edoakes offline. We removed `SENTINEL.VALUE`. Instead, the user can use `StopIteration` or `StopAsyncIteration`.